proj/src/middleware/projectsMiddleware.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+const projectsMiddleware = ({dispatch}) => next => action => {
+  if (!action.meta || action.meta.type !== 'api') {
+    console.log('going to next')
+    return next(action);
+  }
+  
+  axios.get(action.meta.url)
```

Comment:
what if I want a post? or delete?

Reply:
This task includes fetch only (get), the next one is with firebase with a full REST api.